hbw/app/javascript/packs/hbw/components/form/select_table.js.jsx

```diff
@@ -1,154 +1,174 @@
-      const rowParams = this.props.params.row_params;
-      const result = [];
+      onClick(event) {
+        if (this.props.params.editable === false) {
+          return;
+        }
 
-      [...rowParams].forEach((param) => {
-        result.push(<th className={this.buildCssFromConfig(param)} key={param.name}>{param.name}</th>);
-      });
+        this.setState({value: event.target.parentElement.getElementsByTagName('input')[0].value});
+      },
 
-      return result;
-    },
+      buildTableBody(choices, name, value) {
+        const opts = {
+          onClick: this.onClick
+        };
 
-    onClick (event) {
-      if (this.props.params.editable === false) {
-        return;
-      }
+        const cssClassesList = [];
 
-      this.setState({ value: event.target.parentElement.getElementsByTagName('input')[0].value });
-    },
+        Object.keys(this.props.params.row_params).forEach((_) => {
+          const config = this.props.params.row_params[_];
 
-    buildTableBody (choices, name, value) {
-      const opts = {
-        onClick: this.onClick
-      };
+          cssClassesList.push(this.buildCssFromConfig(config));
+        });
 
-      const cssClassesList = [];
+        return choices.map((items) => {
+          let selected;
+          const renderCells = (_items) => {
+            const result = [];
 
-      Object.keys(this.props.params.row_params).forEach((_) => {
-        const config = this.props.params.row_params[_];
+            Object.keys(_items).forEach((i) => {
+              result.push(<td className={cssClassesList[i]} key={i}>{_items[i]}</td>);
+            });
 
-        cssClassesList.push(this.buildCssFromConfig(config));
-      });
+            return result;
+          };
 
-      return choices.map((items) => {
-        let selected;
-        const renderCells = (_items) => {
-          const result = [];
+          const id = items[0];
 
-          Object.keys(_items).forEach((i) => {
-            result.push(<td className={cssClassesList[i]} key={i}>{_items[i]}</td>);
-          });
+          if (this.props.isEqual(id, value)) {
+            selected = 'selected';
+          }
 
-          return result;
-        };
+          const stub = {
+            onChange() {
+            }
+          };
+
+          const isEqualFunc = this.props.isEqual;
 
-        const id = items[0];
+          return <tr {...opts} className={selected} key={id}>
+            <td className='hidden' key={`td-${id}`}>
+              <input {...stub} name={name} type="radio" value={id} id={id} checked={isEqualFunc(id, value)}/>
+            </td>
+            {renderCells(items.slice(1, items.length))}
+          </tr>;
+        });
+      },
 
-        if (this.isEqual(id, value)) {
-          selected = 'selected';
+      buildCssFromConfig(config) {
+        return `text-align-${config.alignment}`;
+      },
+
+      getChoices () {
+        let choices;
+
+        if (this.props.params.mode === 'select') {
```

Comment:
Does select_table have "lookup" type? 
I moved this from mixin here and removed extra option "lookup", as I think.